sklearn/dummy.py

```diff
@@ -304,7 +304,7 @@ def predict_proba(self, X):
         Returns
         -------
         P : ndarray of shape (n_samples, n_classes) or list of such arrays
-            Returns the probability of the sample for each class in
+            Return the probability of the sample for each class in
```

Comment:
No need for this change. This change is only required when this is the summary of the method like for the change that you did for the method `score`.

```suggestion
            Returns the probability of the sample for each class in
```

Reply:
Good to know! Thanks for the advice!